keyboards/senselessclay/had60/had60.h

```diff
@@ -0,0 +1,35 @@
+/* Copyright 2020 Hadi Iskandarani
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+#define ____ KC_NO
+
+#define LAYOUT( \
```

Comment:
```suggestion
#define LAYOUT_all( \
```

Also iso/ansi LAYOUT macros should be added to remove the need to manually pad missing keys with KC_TRANS

Reply:
Made the suggested change, but having separate layouts is only a concern for the configurator, right? All positions within the matrix are accessible here, and some other merged boards also use a single configuration file despite having similar support.